### DISCUSSION TITLE
Serenity BDD using Maven Ecllipse Execution

### DISCUSSION THREAD

**Discussion by Swati-kumari-1** [serenity-bdd/serenity-core#2636] [Q&A] [answered]
I have one doubt , How can I 
Create a script to make get request to the service using rest assured
http://restcountries.eu/rest/v1/name/INDIA
Verify that “
Republic of India”
is found in the response.
Print
that response on the command line.
When you pass country code as xyz
{"status":404,"message":"Not Found"}
is present in response
Print that response on the command line.

**Comment by wakaleo** [comment]
Why do you want to print the response on the command line?

**Reply by Swati-kumari-1** [reply]
I want to print the response through command line or through ecllipse, please guide me the steps via both .

**Comment by wakaleo** [comment, marked as answer]
```
    @Test
    public void shouldReadCountryDataFromAnExistingCountry() {
        Response response = SerenityRest.get("https://restcountries.com/v3.1/name/india");
        System.out.println(response.prettyPrint());
        assertThat(response.jsonPath().getString("name.official[0]")).isEqualTo("Republic of India");
    }

    @Test
    public void shouldReturn404IfNoMatchingCountryIsFound() {
        Response response = SerenityRest.get("https://restcountries.com/v3.1/name/xyx");
        System.out.println(response.prettyPrint());
        assertThat(response.statusCode()).isEqualTo(404);
        assertThat(response.jsonPath().getString("message")).isEqualTo("Not Found");
    }
```

But printing the test results to the console is a bit of an anti-pattern - any test that needs manual verification is not an automated test.

**Reply by Swati-kumari-1** [reply]
Thank u so much

**Reply by Swati-kumari-1** [reply]
> ```
>     @Test
>     public void shouldReadCountryDataFromAnExistingCountry() {
>         Response response = SerenityRest.get("https://restcountries.com/v3.1/name/india");
>         System.out.println(response.prettyPrint());
>         assertThat(response.jsonPath().getString("name.official[0]")).isEqualTo("Republic of India");
>     }
> 
>     @Test
>     public void shouldReturn404IfNoMatchingCountryIsFound() {
>         Response response = SerenityRest.get("https://restcountries.com/v3.1/name/xyx");
>         System.out.println(response.prettyPrint());
>         assertThat(response.statusCode()).isEqualTo(404);
>         assertThat(response.jsonPath().getString("message")).isEqualTo("Not Found");
>     }
> ```
> 
> But printing the test results to the console is a bit of an anti-pattern - any test that needs manual verification is not an automated test.

Thanks for your reply, but I am getting error for assertThat after adding import static org.junit.Assert.assertThat; and for SerenityRest. Could you please guide me the steps.
